src/ConfigurationFactory.php

```diff
@@ -41,6 +41,7 @@ class ConfigurationFactory
     const DEFAULT_KEY = self::ENV_DEV;
 
     const ENV_DEV = 'dev';
+    const ENV_INTEGRATION = 'integration';
```

Comment:
I really hate this ENVs here. I can hardly accept `test`, `live` and `dev`, maybe `integration`. But `uat-sleeper` here is something really wrong.